ZOSOpenTools/bashport
# Bash dev - test case fixes + signal OC4 patch

Test fixes:
* /etc/passwd does not exist on z/OS, add a condition to check for its presence
* Change /etc/passwd to /etc/profile which exists on all platforms
* Use grep -E instead of egrep to avoid "egrep is obsolete" message
* Use /bin/test to check for /dev/stdin because bash emulates /dev/stdin and test builtin thinks it is present
* Adds sig.c patch for Jenkins

Also, we wants the nodir version of /dev/fd to take precendence since all z/OS systems have it

I'll work with Chet from Bash to get these upstreamed.

Should bring down the number of failures to 15
